addons/portal_rating/views/rating_templates.xml

```diff
@@ -39,6 +39,23 @@
             </t>
         </div>
     </template>
+    <template id="rating_widget_stars_static_without_count" name="Rating: static star widget without count">
```

Comment:
Can't we change the id of this template? what do you think about `rating_stars_no_count`?